### ISSUE TITLE
Data inspector text boxes lack bottom focus outline

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-hexeditor#231]
<img width="308" alt="Screen Shot 2021-03-13 at 2 03 38 PM" src="https://user-images.githubusercontent.com/2193314/111051010-e9ab8600-8404-11eb-9730-e5ba1b1bc260.png">
